homeworks/magurmarian/hw01/test_calculation.py

```diff
@@ -0,0 +1,40 @@
+"""module for testing add and devide functions"""
+import argparse
+import sys
+
+from pyats.aetest import Testcase, test, main
+from calculation import add, divide
+
+
+class SmokeTest(Testcase):
+    """class for smoke testing add and devide functions"""
+    @test
+    def test_add(self, num1, num2):
+        """method for testing add function"""
+        if add(num1, num2) < 0:
+            self.skipped('Result is less then 0)')
+
+    @test
+    def test_devide(self, steps, num1, num2):
+        """method for testing devide function"""
+        with steps.start('Divide') as step:
+            try:
+                assert divide(num1, num2) > 0
+            except ZeroDivisionError:
+                step.passx(f'num1:{num1} ,num2:{num2}',
+                           from_exception=ZeroDivisionError('ZeroDivisionError'))
+            except AssertionError:
+                step.skipped('Result is less then 0)')
+
+
+if __name__ == '__main__':
+    parameters = {
```

Comment:
Move "parameters" above classes so it is usable in case of importing this module.
You can also add key values as parser's default parameters.

Reply:
Done!